content/en/docs/tasks/job/parallel-processing-expansion.md

```diff
@@ -1,50 +1,62 @@
 ---
 title: Parallel Processing using Expansions
-content_template: templates/concept
+content_template: templates/task
 weight: 20
 ---
 
 {{% capture overview %}}
 
-In this example, we will run multiple Kubernetes Jobs created from
-a common template.  You may want to be familiar with the basic,
-non-parallel, use of [Jobs](/docs/concepts/workloads/controllers/jobs-run-to-completion/) first.
+This task demonstrates running multiple {{< glossary_tooltip text="Jobs" term_id="job" >}}
+based a common template. You can use this approach to process batches of work in
+parallel.
 
+For this example there are only three items: [_apple_, _banana_, _cherry_].
+The sample Jobs process each item simply by printing a string then pausing.
```

Comment:
nit: do you need the square brackets?

Reply:
Ya. Got to be careful with square brackets ... they are interpreted as out-of-band links in markdown.